requirements.txt

```diff
@@ -1,7 +1,7 @@
-cachetools==5.0.0
-jinja2>=3.0.0
-netaddr>=0.8.0
+cachetools~=5.0
+jinja2~=3.0
+netaddr~=0.8
 pyjwt[crypto]>=1.6.0
-pyyaml>=5.1
+pyyaml~=5.1
```

Comment:
Pyyaml is on 6.0 now. They don't exactly follow semver so >= might make more sense. You could try and research into it more to figure out how their version scheme actually works: https://pypi.org/project/PyYAML/#history

Reply:
So it looks like they use something maybe called "pre-release" versioning. According to [PEP 440](https://peps.python.org/pep-0440/#compatible-release) we can still use `~=`.
> If a pre-release, post-release or developmental release is named in a compatible release clause as V.N.suffix, then the suffix is ignored when determining the required prefix match

Reply:
And I guess it's not *technically* [SemVer](https://semver.org/#spec-item-9)

Reply:
They do that and they don't seem to do `.0` releases. They have `6.0` and `6.0.1`. There is no `6.0.0` which is actually my main concern with ~= compatibility.

Reply:
Not sure how to test that so if you think `>=` is just a safer option then I'll set it back to `>=`.

Reply:
I would have to do more research to give you a conclusive answer, but I'm not working the ticket so I was hoping you'd be able to take a look at that.